src/server/req_runjob.c

```diff
@@ -1377,19 +1377,29 @@ post_sendmom(struct work_task *pwt)
 				        jobp->ji_qs.ji_destin, pbs_errno, reject_msg);
 			else
 				sprintf(log_buffer,
-				        "send of job to %s failed error = %d",
-				        jobp->ji_qs.ji_destin, pbs_errno);
+					"send of job to %s failed error = %d",
+					jobp->ji_qs.ji_destin, pbs_errno);
+
 			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO, jobp->ji_qs.ji_jobid, log_buffer);
 			snprintf(log_buffer, LOG_BUF_SIZE,
 					"Not Running: PBS Error: %s", pbse_to_txt(PBSE_MOMREJECT));
-			if (jobp->ji_qs.ji_svrflags & JOB_SVFLG_SubJob)
-				job_attr_def[(int) JOB_ATR_Comment].at_decode(
+
+			/* job array case */
+			if (jobp->ji_qs.ji_svrflags & JOB_SVFLG_SubJob) {
+				/* if the job array is in state 'B' then its comment should not
+					be modified */
```

Comment:
There are several styles for multi-line comments in the code already.  This is a new one.  I'd choose one of the existing ones for consistency or just make this comment one line.  The most common multi-line comment style is similar to the doxygen header style of starting off with a /* and having *'s down the left hand side ending with a */ on its own line.

Reply:
Fixed.